JS-Algorithms/challenges.js

```diff
@@ -118,7 +168,19 @@ Because the 12th index in the Fibonacci sequence is 144, and 144 has three digit
 ***** */
 
 const fibIndex = (n) => {
-  // YOUR CODE HERE...
+  let prevFibo = 0;
+  let currentFibo = 1;
+  let nextFibonacci;
+  let index = 0;
+
+  for (; prevFibo.toString().length < n; index++) {
```

Comment:
Why do we need to convert prevFibo to string on every iteration?

Reply:
Since I'm not interested to take the integer value as it is, I want the length, so that I stop the execution when a fibo sequence matches the n arg provided in the function. If I don't do that it will be compared with the value itself as integer. And to use length function it needs to be parsed as string